frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -200,6 +203,19 @@ const GraphicalSelector = () => {
     }
   };
 
+  const zoomIn = () => {
+    if (!graph) return;
+    const zoom = graph.getZoom();
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
prefer not disabling eslint line here. You can avoid it by using if statement instead!

Reply:
igtm